Documentation/ceph-pool-crd.md

```diff
@@ -106,6 +106,52 @@ external-cluster-console# rbd mirror pool peer bootstrap import <token file path
 
 See the official rbd mirror documentation on [how to add a bootstrap peer](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#bootstrap-peers).
 
+### Stretched
+
+Imagine the following topology:
+
+```text
+.
+├── datacenter-1
+│   ├── rack-1
+│   │   ├── host-1
+│   │   ├── host-2
+│   │   ├── host-3
+│   │   └── host-4
+│   └── rack-2
+│       ├── host-1
+│       ├── host-2
+│       ├── host-3
+│       └── host-4
+└── datacenter-2
+    ├── rack-1
+    │   ├── host-1
```

Comment:
How about unique host names across racks? CRUSH doesn't allow the same nodes in multiple racks, right?